pipeline/ingestion/src/test/java/org/datacommons/ingestion/data/CacheReaderTest.java

```diff
@@ -126,9 +126,52 @@ public void testParseTimeSeriesRow() {
                 .importName("NOAA_GFS_WeatherForecast")
                 .build());
 
+    NodesEdges expectedGraph = new NodesEdges()
+            .addNode(
+                    Node.builder()
+                            .subjectId("dc/os/Mean_PrecipitableWater_Atmosphere_geoId_sch2915390_870755143")
+                            .name("Mean Precipitable Water Atmosphere | geoId/sch2915390 | 870755143")
```

Comment:
This is fine for now, but maybe we can think about how to make these more readable, especially the facet ids (or if we need the names at all)